docs/markdown/maintenance/upgrading.md

```diff
@@ -1,7 +1,7 @@
 # Upgrading
 
 ## 1 Upgrade Paths
-LME is currently on v1.0.
+LME is currently on v1.x.x.

```

Comment:
What do you think about linking to the releases page here? That way its clear what the latest release of LME is and we don't have to re-write this everytime?

Reply:
I can do that.  I created a task to update the docs in a way so that we take out any hardcoded references to current release numbers.